tests/unit/helper.py

```diff
@@ -79,12 +79,8 @@ def create_instance_of_described_class(self):
         If cls.example_data is None, just create a simple instance of the
         class.
         """
-        if self.example_data:
-            instance = self.described_class(self.example_data,
-                                            self.session)
-        else:
-            instance = self.described_class()
-            instance.session = self.session
+        instance = self.described_class(self.example_data)
```

Comment:
As it was originally, when using it to build a GitHubEnterprise object through create_instance_of_described_class, the GitHubEnterprise object did not have a session set. As a result, a 401 Exception (authentication needed) was thrown during the tests, because create_user requires authentication. 
To solve this, I used to override the create_instance_of_described_class function in the class TestGitHubEnterprise to behave correctly (i.e. take example_data AND set the session at the same time). I can set it like it was if necessary